docs/topics/installing-cli-tool.adoc

```diff
@@ -46,35 +33,143 @@ When you encounter `<{ProductShortName}_HOME>` in this guide, replace it with th
 [id="installing-using-podman_{context}"]
 == Installing the {CLINameTitle} using Podman
 
+You can install the {CLINameTitle} using `podman pull`. This is the recommended approach, except when migrating applications built with a .NET framework on Windows where Docker is required.
+
 .Prerequisites
 
-* Red Hat Container Registry Authentication for `registry.redhat.io`. Red Hat distributes container images from registry.redhat.io, which requires authentication. For more details, see link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
+* Red Hat Container Registry Authentication for `registry.redhat.io`. Red Hat distributes container images from `registry.redhat.io`, which requires authentication. See link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication] for additional details.
 
+* Podman must be installed.
 
-.Procedure
+[NOTE]
+.Podman
+====
+Podman is a daemonless, open source, Linux-native tool designed to make it easy to find, run, build, share, and deploy applications using Open Containers Initiative (OCI) Containers and Container Images.
+Podman provides a command-line interface (CLI) familiar to anyone who has used the Docker Container Engine.
+For more information on installing and using Podman, see link:https://podman.io/docs/installation[Podman installation instructions].
+====
 
-To install using `podman pull`:
+.Procedure
 
-. To use Podman to authenticate to registry.redhat.io:
+. Use Podman to authenticate to `registry.redhat.io` by running the following command:
++
+[source,terminal]
+----
+$ podman login registry.redhat.io
+----
++
+Enter the user name and password:
 +
 [source,terminal]
 ----
-podman login registry.redhat.io
 Username: <username>
 Password: <***********>
 ----
-. Issue:
+
+. Copy the binary `PATH` to enable system-wide use by running the following command:
 +
 [source,terminal]
 ----
-podman cp $(podman create registry.redhat.com/mta-toolkit/mta-mta-cli-rhel9:{ProductVersion}):/usr/local/bin/mta-cli ./
+$ podman cp $(podman create registry.redhat.com/mta-toolkit/mta-mta-cli-rhel9:{ProductVersion}):/usr/local/bin/mta-cli ./
 ----
 +
-This command will copy the binary `PATH` for system-wide use.
-+
 [WARNING]
 ====
 Although installation using Podman is possible, downloading and installing the `.zip` file is the preferred installation.
 ====
 
+[id="installing-using-docker_{context}"]
+== Installing the {CLINameTitle} for use with Docker on Windows
+
+You can install the {CLINameTitle} for use with Docker on Windows. This is the required approach when migrating applications built with .NET framework 4.5 or later on Windows to cross-platform .NET 8.0.
```

Comment:
Shouldn't we also mention in this section the need to override the environment variable `PODMAN_BIN` to force mta_cli to use Docker and `DOTNET_PROVIDER_IMG` to point to the image for windows/amd64?

@dymurray @djzager

Reply:
This is using PowerShell, not Bash.

Reply:
Hi @johnwilkins, yes, I'm referring to PowerShell as well. 

I think we should document that the user has to add those 2 Windows ENV variables, but I'd like @dymurray, @djzager confirmation on that.

Reply:
This is true. Assuming podman is already installed, you must set `$env:PODMAN_BIN="C:\Windows\system32\docker.exe"`. When analyzing .NET Framework applications, you'll need to override the provider image (because only upstream has windows/amd64 images); `$env:DOTNET_PROVIDER_IMG="quay.io/konveyor/dotnet-external-provider:v0.5.0"` for example.

Reply:
Come to think of it, you'll also need to override the RUNNER_IMG for the same reason. `$env:RUNNER_IMG="quay.io/konveyor/kantra:v0.5.0"`

Reply:
Fixed.